radar/radar.go

```diff
@@ -92,6 +108,8 @@ func (radar *Radar) Scanner(logger lager.Logger, resourceName string) ifrit.Runn
 				if err != nil {
 					return err
 				}
+
+				timer.Reset(radar.interval)
```

Comment:
@vito - you are totally right.  I thought you were crazy but I went back and read the docs again.  Don't know how I missed that :/